storage.bs

```diff
@@ -211,6 +228,28 @@ It is initially empty.
 browsing session. It is roughly analogous to <a>top-level browsing context</a> except that it cannot
 be replaced due to <code>Cross-Origin-Opener-Policy</code> or navigation.
 
+<p>To <dfn export>legacy-clone a browsing session storage shed</dfn>, given a browsing session
+<var>A</var> and a browsing session <var>B</var>, run these steps:
+
+<ol>
+ <li>
+  <p><a for=map>For each</a> <var>key</var> → <var>shelf</var> of <var>A</var>'s
+  <a for="browsing session">storage shed</a>:
+
+  <ol>
+   <li><p>Let <var>newShelf</var> be the result of running <a>create a storage shelf</a> with
+   "<code>session</code>".
+
+   <li><p>Set <var>newShelf</var>'s <a>bucket map</a>["<code>default</code>"]'s
+   <a>bottle map</a>["<code>sessionStorage</code>"]'s <a for="storage bottle">map</a> to a
+   <a for=map>clone</a> of <var>shelf</var>'s <a>bucket map</a>["<code>default</code>"]'s
+   <a>bottle map</a>["<code>sessionStorage</code>"]'s <a for="storage bottle">map</a>.
+
+   <li><p>Set <var>B</var>'s <a for="browsing session">storage shed</a>[<var>key</var>] to
+   <var>newShelf</var>.
+  </ol>
+</ol>
```

Comment:
It'd be good to add a note explaining why this is legacy. The algorithm seems very sensible and generally applicable.